src/scenes/Partnerships/Create/CreatePartnership.tsx

```diff
@@ -43,6 +44,7 @@ export const CreatePartnership = ({
         (res) => res.createPartnership.partnership
       )
     ),
+    refetchQueries: [GQLOperations.Query.ProjectPartnerships],
```

Comment:
I'm trying to avoid refetch queries. It's much faster to update the cache directly. We have a handful of mutation `update` functions across the app now. Take a look at those and take a stab at just finding an old primary partnership and setting it to false.